doc/source/conf.py

```diff
@@ -171,6 +171,14 @@ def setup(app):
 
 html_favicon = '_static/favicon/favicon.ico'
 
+# Set up the version switcher.  The versions.json is stored in the devdocs.
+if ".dev" in version:
+    switcher_version = "devdocs"
+    switcher_json_url = "_static/versions.json"
```

Comment:
Hmmm, this only works on the landing page correctly.  Should we just always use `"https://numpy.org/devdocs/_static/versions.json"`?

Reply:
nvm, it may make sense, but this is not right anyway, and to really get it right, it should probably recognize a proper deploy from e.g. a PR.  So just using the full url.

This should work as soon as it is deployed, although of course `numpy.org/devdocs/_static/versions.json` would need to remain available.  Pandas put this elsewhere on their website I think, but then they build the website in the main repo?